### PR TITLE
Migration: Migrated RothCOutput.vue to Vue 3 (Composition API) 

### LINKED ISSUE #338: migration: migrate RothCOutput.vue to Vue 3 (Composition API)
- Migrate the [RothCOutput.vue file](https://github.com/moja-global/FLINT-UI/blob/master/flint.ui/src/views/flint/RothCOutput.vue) in src/views/flint/ to use CompositionAPI (check out https://github.com/moja-global/FLINT-UI/pull/311 PR by @Palaksharma23, or any other migration PR for reference.).
- Rename RothCOutput.vue to RothCOutputGraph.vue to make the name more descriptive.

If you have any questions or need help please reach out to me (or any other community member) here or on Slack.

### PR DESCRIPTION
**Pull request by legendarykamal**
# Pull Request Template

## Description

Migration: Migrated RothCOutput.vue to Vue 3 (Composition API) 

This PR fixes #338 
## Testing

Please instructions so we can verify your changes.

If our automated tests fail, please feel free to reach out to us by leaving a comment down below and we will be happy to take a look

## Additional Context (Please include any Screenshots/gifs if relevant)

...

### CODE REVIEW

**Comment by legendarykamal** [comment]
@YashKandalkar Please review :smile:

**Comment by chicken-biryani** [comment]
@Palaksharma23 can you please take a look as well?

**Comment by Palaksharma23** [comment]
Hey! @legendarykamal, can you please work on solving the deployment errors

**Comment by legendarykamal** [comment]
@HarshCasper @chicken-biryani anything you can suggest so that I can debug this ?

**Comment by chicken-biryani** [comment]
Hey @legendarykamal feel free to reach out to @Palaksharma23 or @Janvi-Thakkar over slack!

**Comment by YashKandalkar** [comment]
@legendarykamal you can have a look at the CI / build run details for errors

**Comment by legendarykamal** [comment]
finally fixed the CI error :)

**Comment by thisiskaransgit** [comment]
hey @legendarykamal are you still working on it

**Comment by legendarykamal** [comment]
yeah on it @thisiskaransgit just have to fix these build errors .

**Comment by legendarykamal** [comment]
I rebased to fix the DCO and made changes to fix the build. Hopefully builds with no issues this time 🤞

**Comment by legendarykamal** [comment]
Wohoo done! all checks green @YashKandalkar @chicken-biryani @HarshCasper  💯

File: flint.ui/src/views/flint/ConfigurationsRothC.vue [hunk 1]
```diff
@@ -79,7 +79,7 @@
 import RothCTemplate from '@/views/flint/RothCTemplate.vue'
 import Datepicker from '@/components/Datepicker/DatepickerRothC.vue'
 import { ButtonComponent } from '@moja-global/mojaglobal-ui'
-import RothCOutputContainer from './RothCOutputContainer.vue'
+import GraphContainer from './GraphContainer.vue'
```
Comment (Crystalsage) [inline flint.ui/src/views/flint/ConfigurationsRothC.vue:82]: @legendarykamal Your build still fails. It's because `GraphContainer.vue` doesn't exist.

**Comment by HarshCasper** [comment]
> Wohoo done! all checks green @YashKandalkar @chicken-biryani @HarshCasper 💯

Where? The Vercel deployment still fails. Please take a look at @Crystalsage's comment and fix the issue he pointed out.

**Comment by Janvi-Thakkar** [comment]
Hello @legendarykamal, Please do let us know if you are still working on resolving the build failures.

**Comment by Janvi-Thakkar** [comment]
Hi @thisiskaransgit! As you were asking, you may continue this.

**Pull request closed without merging**